source/features/sticky-file-header.css

```diff
@@ -0,0 +1,16 @@
+/* `.sticky-file-header` excludes native sticky header */
+.file-header:not(.sticky-file-header) {
+	position: sticky;
+	top: 0;
+	/* Same as `.sticky-file-header.is-stuck */
+	z-index: 6;
+}
+
+/* Same as `.sticky-file-header.has-open-dropdown` */
+.file-header.has-open-dropdown:not(.sticky-file-header) {
+	z-index: 10;
+}
```

Comment:
Would it break something if we set it to 10 by default?

Reply:
I don't want to mess with z-index, make it always 6 is already a bit 😬